packages/vue/src/components/molecules/SfSelect/SfSelect.vue

```diff
@@ -48,6 +47,7 @@
           <ul
             :aria-expanded="open.toString()"
```

Comment:
all of aria label should move to `ul` since it's about the select dropdown. Also why not use `select` instead of `ul`?

Reply:
Thanks for review @mayashavin! Regarding why not use a select tag, as far as I know, styling native elements of select is hard especially with our design, so in this case, I'd stay with our ul and li solution. If you have a solution for this, let me know :)